fulp_modules/ipc/code/human_defense.dm

```diff
@@ -0,0 +1,34 @@
+/mob/living/carbon/human/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_CONTENTS)
+		return
+	var/informed = FALSE
+	var/affects_leg = FALSE
+	var/stun_time = 0
+	for(var/obj/item/bodypart/L in src.bodyparts)
+		if(L.status == BODYPART_ROBOTIC)
+			if(!informed)
+				to_chat(src, "<span class='userdanger'>You feel a sharp pain as your robotic limbs overload.</span>")
+				informed = TRUE
+			switch(severity)
+				if(1)
+					L.receive_damage(0,6)
+					stun_time += 40
+				if(2)
+					L.receive_damage(0,3)
```

Comment:
does this damage pierce the damage reduction of augged limbs, or does it just get no-selled by them?